cmd/gossamer/setup.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2023 ChainSafe Systems (ON)
```

Comment:
maybe merge this with main.go? Don't really see the need for seperate files.

Reply:
The intention behind it was to include the ENVs in `setup.go` at a later stage. It could make things messy if we keep everything in `main.go`.

Reply:
Ended up removing few unused functions and merged it to main.go